CHANGELOG.md

```diff
@@ -5,13 +5,47 @@ All notable changes to Zebra are documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [Zebra 1.1.0](https://github.com/ZcashFoundation/zebra/releases/tag/v1.1.0) - 2023-07-18
 
-## [Zebra 1.0.1](https://github.com/ZcashFoundation/zebra/releases/tag/v1.0.1) - 2023-XX-XX
+Zebra now uses less memory after the final checkpoint by deduplicating note commitment trees in the non-finalized state.
+
+This release also adds new mempool metrics, fixes panics when cancelling tasks on shutdown, and improves the usability of Zebra's Docker images (particularly for mining).
 
 ### Breaking Changes
 
 - Zebra now detects subcommand name typos on the command-line. If you want to give Zebra a list of tracing filters, use `zebrad start --filters debug,...` ([#7056](https://github.com/ZcashFoundation/zebra/pull/7056))
```

Comment:
Optional: This might be worth a brief mention in the intro, because it might impact a few users.

Reply:
Added in https://github.com/ZcashFoundation/zebra/pull/7259/commits/8186212f4bfc5af1311aa99ba8d57abbb80f550d.